tools/csv-merger/csv-merger.go

```diff
@@ -90,18 +91,21 @@ var (
 	passtCNIImage       = flag.String("network-passt-binding-cni-image-name", "", "Passt binding cni image")
 	waspAgentImage      = flag.String("wasp-agent-image-name", "", "Wasp Agent image")
 	smbios              = flag.String("smbios", "", "Custom SMBIOS string for KubeVirt ConfigMap")
+	smbiosFile          = flag.String("smbios-file", "", "Custom SMBIOS file name, for KubeVirt ConfigMap")
```

Comment:
for a long time there is no kubevirt config map. all kubevirt configurations are in the kubevirt CR. maybe it worths to update the description to avoid confusion?